alarm/kodi-rbp/PKGBUILD

```diff
@@ -23,138 +24,113 @@ makedepends=('hicolor-icon-theme' 'fribidi' 'lzo2' 'smbclient' 'libtiff' 'libva'
-           'libplist' 'swig' 'taglib' 'libxslt' 'shairplay' 'libdcadec.so')
+           'libmpeg2' 'libmad' 'libmodplug' 'jasper' 'rtmpdump' 'xorg-xdpyinfo' 'libbluray'
+           'avahi' 'bluez-libs' 'tinyxml' 'raspberrypi-firmware' 'libpulse'
+           'libplist' 'swig' 'taglib' 'libxslt' 'libcrossguid-git')
 
   optdepends=(
-    'afpfs-ng: Apple shares support'
+    'afpfs-ng: Apple airplay and AFP share support'
+    'libcec-rpi: Pulse-Eight USB-CEC adapter support'
+    'libnfs: NFS shares support'
     'lirc: remote controller support'
+    'lsb-release: log distro information in crashlog'
     'polkit: permissions for automounting external drives and power management functionality'
-    'udisks: automount external drives'
-    'upower: used to trigger power management functionality'
+    'shairplay: limited Apple airplay support'
     'unrar: access compressed files without unpacking them'
-    'lsb-release: log distro information in crashlog'
-    'gdb: stack trace in crashlog')
+    'unzip: access compressed files without unpacking them'
+    'upower: used to trigger power management functionality')
 
-  install="kodi.install"
+  install='kodi.install'
   provides=('xbmc' 'kodi')
   conflicts=('xbmc' 'kodi' 'arm-mem-git' 'shairplay-git')
-  replaces=('xbmc-rbp')
+  replaces=('xbmc-rbp-git')
 
-  cd "$srcdir/xbmc-$pkgver-$_codename"
-  # Running make install
-  make DESTDIR="${pkgdir}" install
+  _components=('kodi' 'kodi-bin')
 
-  # We will no longer support the xbmc name
-  rm "$pkgdir/usr/share/xsessions/xbmc.desktop"
-  rm "$pkgdir/usr/bin/"xbmc{,-standalone}
-  # we will leave /usr/{include,lib,share}/xbmc for now
+  cd kodi-build
+  # install eventclients
+  for _cmp in ${_components[@]}; do
+  DESTDIR="$pkgdir" /usr/bin/cmake \
+    -DCMAKE_INSTALL_COMPONENT="$_cmp" \
+     -P cmake_install.cmake
+  done
 
   # Licenses
-  install -d -m 0755 "${pkgdir}${_prefix}/share/licenses/${pkgname}"
+  install -dm755 ${pkgdir}/usr/share/licenses/${pkgname}
   for licensef in LICENSE.GPL copying.txt; do
-    mv "${pkgdir}${_prefix}/share/doc/kodi/${licensef}" "${pkgdir}${_prefix}/share/licenses/${pkgname}"
+    mv ${pkgdir}/usr/share/doc/kodi/${licensef} \
+      ${pkgdir}/usr/share/licenses/${pkgname}
   done
 
-  install -Dm0644 $srcdir/kodi.service $pkgdir/usr/lib/systemd/system/kodi.service
-  mkdir -m700 -p $pkgdir/usr/share/polkit-1/rules.d/
-  install -Dm0644 $srcdir/polkit.rules $pkgdir/usr/share/polkit-1/rules.d/10-kodi.rules
-  chmod 0750 $pkgdir/usr/share/polkit-1/rules.d/
+  # python2 is being used
+  cd "$pkgdir"
+  grep -lR '#!.*python' * | while read file; do sed -s 's/\(#!.*python\)/\12/g' -i "$file"; done
+
+  install -Dm0644 "$srcdir/kodi.service" "$pkgdir/usr/lib/systemd/system/kodi.service"
+  install -Dm0644 "$srcdir/polkit.rules" "$pkgdir/usr/share/polkit-1/rules.d/10-kodi.rules"
+  chmod 0700 "$pkgdir/usr/share/polkit-1/rules.d/"
```

Comment:
Should this be `0750`? [`polkit` says `0750`](https://github.com/archlinuxarm/PKGBUILDs/blob/417b9664c5a3a96663fb93fc6f521e85c2242299/extra/polkit/PKGBUILD#L54).